01-Fundamentos/Bloco-03-Introducao-a-HTML-e-CSS/3.01_Introducao_a_HTML_e_CSS/index.html

```diff
@@ -0,0 +1,34 @@
uqiN2k7bFBV0rDvWGNkdD4CvIyVka4r+yHBPmzOUswAGq2IFjgk9D+vExDl3WiyncXfy4+c0sTHZCNSgqpD3VNR+8SKvfz8pcTH1oWARlVVNmr2qzudqqQ80l0XBKxRfV8nOamoVqsNVdN7JI8OAIuAGal8VbgX90Eb7Hbeps4roymkXcGihHNXekb3vwfP0lEYqAkVSkAiidibvbkbgbyVmtPgh40u5S9wOur5p/SJNz7bg9ME1076jbpt0hI96Xhj2o+Uch2vmkc0BrRFIDDZ7XTq1Fhv3iN6O0xGwwTq20k7Dg/W9rkumwaHdG7C6/R248pHl8QA7ixXBqjvuOmna/jPVxw0VLsebOW0rZLjMuvuigATXmaH4Dz9TFymH12+9Xw899vpJUVARqPeFnTSsrCrA1C/LvefkYxFAGkHc77WSdibIG46j5dN5ZvikRFU7ZZV9PeOlCpWt7JN9RZr1+ks9oF9SML74UDpvVD0Jq+OvWZGJY3ujRNDpfQ+B8pZXOP7vgWrABqBItSBV+S/QfHJwdpo0WTqmVMbHZcRirEW1mtgbNmx4b8R+axGd2I5LbAdBbABQOOOI3BwtZ717+Hy+c0xlVYXoICqdlALcBt7N3pP023M0k1GrM4xcroXK4bIh96XAO6rZ3Iq2roBYsnxEt5FCxrDY2p5ZaUVRO5DbWOo6GMbKOigFdxuSVsCwAB3h96trXrXrJEw3UEe7YKGNWhI4NrR26A9PvcePPJ7W7OmK1pUX8TFzJY3h4bgGy2hLVvEUATvxV9Jy/amO7uS7lyBVkAHbkbeHF+U2sbtBk198nekGkhQSD3gejUTtKX2QBgXDqLBbmxdldyNr2O/g3lGJau2l/Ay/JUmyrkMVE2ZQbINspIrrQA/VSPP4jO5YitgAKoAKOAo4HM18XNIjsE1KF0kMdKtq3JIDGx0A0mZiKXYkk7m2aiQLO5NTWLt7UZSjS1sgLna2JoADe+6BQHwGwHSqjzjEoE6Bi3xoAceVyxnnwi/cOoUBsukbCuNrO2/iblcstCtvHa63l07SdFWqbVl/sftZsIlFI0PySKKnoytsRwL3ogVJsx7R46lkw8RghWtqrpqK86bI6GU+y1BxKKK4okjQGFcG7IA559JN2xhW+ohgNxulAVwFazqFHk+cy1h7nTmjRSlpwyLIZ4+9RnN7m7APII4rfc8Tcwu2cW9QcAE7akLHUTsHIJVN6P7PPFVOVRqO/ToRsam2j4bg6CysKZghar2rWgUqRf8vFHiRlxxbTaLYskkmkyt2li4pY+8cu4O5JsURtvfgeK8pnlb20j8PwmpmssrKrJquqeygI8CyrVHx45HW5W9wNAYBj94HmgVAN36H/bLwkqKTi2xeycT3b7BgWoAq2mje3JAIuufATqFzWPrZlXWQnOvDFbDnu2wuz8OJxykWOo8+vltOjyGnES2NlR3rVRzQpXLWOvPh6zH1EF9ma4ZP6o0cB8TFKguWajTKFJ8FDllIUEb1p/rLWZy2IWVHxfu1di9QG6nTwATQ2GxuvLJGQdQPco41Ha3rSb2qmNgUbrovpdfPrjhtHvDQpSNZG9HcljvfeF+Q3nNps/i0jo2pcpnQ42Zy2GhUFToG+kbiiNRNcHvcjxEz8z2kpB0k6QO9Y72nTZuqGrSx48ONpk5bKP3lVLU7GnsgmwG38dNmr4+Qw9zYWy7AErVgK4+8aFAgeX7XnJjhSfW2Vllb7Uiy+ORqcWLJ22INEGjRsULFUJJgdqKhZmYKp8CNyvNgAkMbu/QgdJhe8KuWBpdV2dt9ug6E3sB/WR5sg6gDY1XzdbnvE9TvX9dpv7CfDMHmaVo6XMdrYLWqMVARhqIu22Zb6m9zz0uQ5nMlbZMTUiitPNnlQoAstuNzXWce1nxIA63sL8egsyRGG1mxvdbdKG3HPzkr0kY9GV/yW+x0OWxveI6kgmySK3LMBuSTQ+7xXQxmLiah1DleAd6FEXe1jewK5+VPA7RcaaJ2O7AncE94eXX85czSMcN8RksbaP2SASdwCSWHBJG9k89Kaay5/glZHJFXEzzKSLG3r/WEpax/wB3+RhN9F4M9xmWIo3W4NbWaJHoPD+0h06CFYDY2a56dYkJsurMn0Q7FzFgVyK3IF7XW/PUyC/lCEukijbL7INKB3oNRNAkgb7+Bq6rb7vxL1y6lHrotjat1FkkWb1C/Q10hCY2apDsmbBCAW2m7/hv6WpP6E6PLsF1Lw21nciyBXXnxbzhCYZzowj8c0hCsRfNkmrUsdPPUck3KuJ2guKDuWYkkE92iGDGqFeP8xhCYwSNZPgxlxfeP3tRVV6m9ht5eI4nS4WEFSg9E77gm/kPPTZ4okDxITTN2K4e5mds5dVUkhQ/dqhRskkmwK3FeHX0mXkMriPYwuas8A0DXJPn0hCaQb9tlMiW5XOBTFTyCPrvv6XDOpoOnvBq7wJBprPBHSq8YQmyfK/Bzvo/ySZfExF+6zKWqiCN63AvkDcS7h4GLiAsznSAA5JLVdVsTud6seMSEpPhmsFaK2bwioq9rJHS7OkEAeXp1mllUUJh0iK7bh9JZjQYdWocXUISs38UWglsyfGzqDWGbSANLWtsxIJ6AgAkk8nfwmRlM6qsyUGBsbg1YNg9P3YkJGOK1YySdofncsFVXG6vZAoCqlfCzJRlI5FVv0/6hCax5jz+5nLiXH7HQ5HttARYxOKC2tbAaj59d7B4je1HAJxFd1I3VTR2NVZHOx3swhOXRKfB07tx5I8btOhp+6aHeAPe2DeJNEePUb81MvN5slqFA93cWD1IB3onfnjceEITbFBWY5ZujOYGxuSSaO/jtX1+ssrkzo1XVnbc7jfy2hCbTbVUYRViY+AxIArjYcbc7nazvEwFVgwOzAdOOSfDwH0+EISLdDuGXVSdIJBY0DQPJAH4x2ZR8O7a6NAHfugirG4A42F9YQkP7In9LEwsurAH3lX0IP5CEIRb8laP/9k=" alt="">
```

Comment:
É legal achar uma URL mais curta pra imagem. As vezes se você procurar pode encontrar a mesma imagem em várias fontes.
```suggestion
    <img src="https://espaciomex.com/wp-content/uploads/2020/04/helena-lopes-PGnqT0rXWLs-unsplash.jpg" alt="Amigos se abraçando" width="400px">
```